### ISSUE TITLE
Add DeepAlpha - Deep learning crypto trading bot

### ISSUE THREAD

**Issue by stefanoviana** [ChristosChristofidis/awesome-deep-learning#305]
Suggesting [DeepAlpha](https://github.com/stefanoviana/deepalpha) for **Projects** section.

AI crypto trading bot using LightGBM + XGBoost ensemble with 72 ML features, HMM regime detection, Temporal Fusion Transformer. 70.9% walk-forward validated accuracy. MIT licensed.

**Comment by aiasistant88-debug** [comment]
привіт взяв на тест вашу модель
Дві години мова мовчить
Мені здається вана взагалі не робоча
Супорт також не відповідає

**Comment by stefanoviana** [comment]
Hello,

  Could you please share:
  - Registered email
  - Screenshot of the issue
  - Exchange you're using

  I'll check your account and get back to you shortly.

  Best regards,
  Stefano

  ==================================================

Il giorno dom 24 mag 2026 alle ore 20:39 aiasistant88-debug <
***@***.***> ha scritto:
